frb_example/pure_dart_multi/rust/external/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "flutter_rust_bridge_example_external"
```

Comment:
Again, no need for this "external" subpackage I guess?

Reply:
Fixed in https://github.com/fzyzcjy/flutter_rust_bridge/commit/3917234ba483bcd16ba702640a8d4b806d27d912.